app/controllers/api/internal/reviews_controller.rb

```diff
@@ -0,0 +1,29 @@
+class Api::Internal::ReviewsController < Api::Internal::BaseController
+  before_action :authenticate_user!
+
+  def index
+    respond_with Review.all
```

Comment:
<sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Not covered lines: 5